### ISSUE TITLE
Details about training

### ISSUE THREAD

**Issue by zyy-cn** [seoungwugoh/ivs-demo#7]
Hi:

 Thanks for sharing this code. Based on this code I'm trying to reproduce the results reported in table.2, which the model is trained only on DAVIS 17. The reported AUC=0.555 and J@60=0.589. However in my reproduction, only AUC=0.496 and J@60=0.521 achieved:

  time:  [0.00, 4.64,  9.63,  14.19, 18.32, 22.38, 26.16, 29.85, 33.67, 102.02]
  J:        [0.00, 39.86, 45.37, 48.06, 49.39, 50.31, 50.93, 51.56, 52.08, 52.08]
	
Since I don't know the difference between my implementation and yours, I present my training details for review:
	
1. data prepare:
	1.1 split each sequence in DAVIS17 into multi new sequence, each new sequence contains only ONE target object. The total number of new training sequences is 144. 
	1.2 calculate current max_skip_step, max_skip_step is linearly increased from 4 to 8. (increase 1 at every 20 epochs and fixed to 8 after 80 epochs)
	1.3 sample 8 frames with a fixed interval according to current skip_step, which is a random integer in [4, max_skip_step]. 
	1.4 data augment
		1.4.1 Resize: resize the shortest edge to 480 and keep the ratio of image.
		1.4.2 RandomCrop: random crop the same 400x400 location in all 8 frames and make sure the new sequence contains the target object.
		1.4.3 RandomAffine: scale=(0.9, 1.1), shear=(-15, 15), rotate=(-25, 25)
		1.4.4 RandomContrast
		1.4.5 AdditiveNoise
		1.4.6 RandomMirror
2. training details:
	2.1  calculate max_num_interaction. (max_num_interaction = 1 if skip_step < 5, max_num_interaction = 2 if skip_step < 7, max_num_interaction = 3 if skip_step >= 7)
	2.2  random select a frame in the first round to interact with scribble based on robot proposed on Davisinteractive framework. In the following rounds, the worst segmented frame will be used.
	2.3  infer the current intermediate estimations based on current scribble, previous estimations and aggregated feature(if available)
	2.4  collect CE losses from multi-scale decoder outputs ((256,256), (64,64), (32,32), (16,16), (8,8)) of interaction-net and propagation-net for all the intermediate estimations and perform back-propagation. 
	2.5  clear all the gradient and the graph before next round. 
	2.6  soft-aggregation post-processing is not performed since the training sample only contains a single target object.
	2.7  the number of training epoch is set to 2000, each epoch has 144 training samples, each sample contains 8 frames, and each frame contains one target object (at most).
	2.8  we use SGD to optimize all the parameters in interaction and propagation networks except all the BN layers. The learning rate is fixed to 5e-5 and momentum = 0.9. The training batch is set to 1. 
	2.9  we initialize the model with the weight pre-trained on ImageNet.
	2.10 we use a single V100 GPU to train and evaluate this model.
	
Are these training details correct?
